### PR TITLE
Fixes #29: Add-Color-palette-to-Website

### LINKED ISSUE #29: [Feature Request]: Add Color palette to Website
### Is your feature request related to a problem? Please describe.

We need proper color palette for our website

### Describe the solution you'd like

Colors from this website: https://fonts.google.com/specimen/Open+Sans

| Color Name | Color Code |
|--------|--------|
| orange | #ff6725 |
| orange-dark | #ff3f25 |
| orange-light | #fef7f3 |
| brown | #453a34 |
| green | #00b579 | 

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by KvngJamesNG**
Update App.jsx
I need your help on this because I have tried 

module.exports = {
  content: ["./src/**/*.{js,jsx,ts,tsx}", "./index.html"],
  theme: {
    extend: {
      colors: {
        orange: '#ff6725',
        'orange-dark': '#ff3f25',
        'orange-light': '#fef7f3',
        brown: '#453a34',
        green: '#00b579',
      },
    },
  },
  plugins: [],
};
 in the tailwind file and 

const App = () => {
  return (
    <div className="p-8">
      <h1 className="text-orange text-4xl font-bold">Welcome to KlinicCon!</h1>
      <p className="text-orange-dark">We use the best medical practices.</p>
    </div>
  )
}

export default App;
In the .jsx file 

@tailwind base;
@tailwind components;
@tailwind utilities;
in the .css file 

but it is not working and I have changed files severally

### CODE REVIEW

**Comment by rt4914** [comment]
@KvngJamesNG Maybe we can meet and discuss this.

Else, refer to this https://stackoverflow.com/a/64175955
https://tailwindcss.com/docs/customizing-colors

**Comment by KvngJamesNG** [comment]
@rt4914 I have few questions to ask.

first of all, thank you for the documentations they were really helpful and also your video about interview with deliver also gave me some wisdom and tips on why not rely on chatgpt sometimes to ask questions.

1. my first question is, apparently I am trying to learning tailwind css bit by bit through usage, but I want to ask is that key and value in tailwind can got in 2 ways. Ex:
colors: {
        'orange': '#ff6725',            ==         orange:  '#ff6725',    ==     orange: #ff6725, (this last one is in some cases it works) 
      },

2. should I generate the color palette myself or we will decide as we keep working on this project.
3.  the link you added on this issue Is not for color palette it is for font style right?

**Comment by rt4914** [comment]
> @rt4914 I have few questions to ask.
> 
> first of all, thank you for the documentations they were really helpful and also your video about interview with deliver also gave me some wisdom and tips on why not rely on chatgpt sometimes to ask questions.
> 
> 1. my first question is, apparently I am trying to learn tailwindcss bit by bit through usage, but I want to ask is that key and value in tailwind can go in many ways.
> 
> Ex:
> 
> colors: { 'orange': '#ff6725', == orange: '#ff6725', == orange: #ff6725, (this last one is in some cases it works) },
> 
> 3. should I generate the color palette myself or we will decide as we keep working on this project.
> 4. the link you added on this issue Is not for color palette it is for font style right?

@KvngJamesNG 

1. The correct way of mentioning colors is `'orange': '#ff6725'` and in some cases this also works `orange: '#ff6725'`. The last one that you mentioned should never work.
2. I have mentioned all the required colors in the issue so you can refer to that and use them in code.
3. The links that I provided are related to color-pallete only.

File: tailwind.config.js [hunk 1]
```diff
@@ -5,7 +5,14 @@ export default {
     "./src/**/*.{js,ts,jsx,tsx}",
   ],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        'orange': '#ff6725',
+        'orange-dark': '#ff3f25',
```
Comment (rt4914) [inline tailwind.config.js:11]: orange-light is missing

File: tailwind.config.js [hunk 2]
```diff
@@ -5,7 +5,14 @@ export default {
     "./src/**/*.{js,ts,jsx,tsx}",
   ],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
```
Comment (rt4914) [inline tailwind.config.js:9]: If you check the resource that I shared https://stackoverflow.com/a/64175955

In that they use first get all the defaultcolors and than add the custom colors. 

In you case you are directly adding custom colors and as a result we won't be able to use any color which tailwind provides by default.

Check the link again and try to apply solution similar to that.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@KvngJamesNG PTAL. Thanks.

**Comment by rt4914** [comment]
@KvngJamesNG Also, please mention issue number in the Pull Request Description and in future follow the template that the pull request follows.
Actually you `main` and this current branch is not up-to-date with latest code. You will need to run commands like this in git-terminal to update your branches.

### Updating local-main branch with the latest remote-main branch
```
git checkout main
git fetch origin
git merge origin/main
```

### Updating current branch with main
```
git checkout kvngjamesng/Add-Color-Palette-to-Website
git fetch origin
git merge origin/main
```

Assuming there are no merge conflicts or any other issues. If you need more help feel free to reach out.

File: tailwind.config.js (hunk 2 above)
Comment (KvngJamesNG) [reply tailwind.config.js:9]: @rt4914 thank you for this observations.

It was an honest mistake.

I will rectify it. 🤝🏾

**Comment by KvngJamesNG** [comment]
> @KvngJamesNG Also, please mention issue number in the Pull Request Description and in future follow the template that the pull request follows.
> 
> Actually you `main` and this current branch is not up-to-date with latest code. You will need to run commands like this in git-terminal to update your branches.
> 
> 
> 
> ### Updating local-main branch with the latest remote-main branch
> 
> ```
> 
> git checkout main
> 
> git fetch origin
> 
> git merge origin/main
> 
> ```
> 
> 
> 
> ### Updating current branch with main
> 
> ```
> 
> git checkout kvngjamesng/Add-Color-Palette-to-Website
> 
> git fetch origin
> 
> git merge origin/main
> 
> ```
> 
> 
> 
> Assuming there are no merge conflicts or any other issues. If you need more help feel free to reach out.
> 
> 
> 
> 

@rt4914 tbh i am having issue with my machines terminal and GitHub on it because it doesn't run git merge or git pull or push commands that well so i usually do it manually(either i download recent version and amend on local then refresh GitHub then paste the changes manually on GitHub remote or I just work on the files on GitHub directly all in my branch).

But overall i will fix this issue too.

Many thanks 😊

**Comment by rt4914** [comment]
> > @KvngJamesNG Also, please mention issue number in the Pull Request Description and in future follow the template that the pull request follows.
> > Actually you `main` and this current branch is not up-to-date with latest code. You will need to run commands like this in git-terminal to update your branches.
> > ### Updating local-main branch with the latest remote-main branch
> > ```
> > 
> > git checkout main
> > 
> > git fetch origin
> > 
> > git merge origin/main
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > ### Updating current branch with main
> > ```
> > 
> > git checkout kvngjamesng/Add-Color-Palette-to-Website
> > 
> > git fetch origin
> > 
> > git merge origin/main
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Assuming there are no merge conflicts or any other issues. If you need more help feel free to reach out.
> 
> @rt4914 tbh i am having issue with my machines terminal and GitHub on it because it doesn't run git merge or git pull or push commands that well so i usually do it manually(either i download recent version and amend on local then refresh GitHub then paste the changes manually on GitHub remote or I just work on the files on GitHub directly all in my branch).
> 
> But overall i will fix this issue too.
> 
> Many thanks 😊

@KvngJamesNG Lets meet and try to fix it.

**Comment by KvngJamesNG** [comment]
> > > @KvngJamesNG Also, please mention issue number in the Pull Request Description and in future follow the template that the pull request follows.
> > > Actually you `main` and this current branch is not up-to-date with latest code. You will need to run commands like this in git-terminal to update your branches.
> > > ### Updating local-main branch with the latest remote-main branch
> > > ```
> > > 
> > > git checkout main
> > > 
> > > git fetch origin
> > > 
> > > git merge origin/main
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > ### Updating current branch with main
> > > ```
> > > 
> > > git checkout kvngjamesng/Add-Color-Palette-to-Website
> > > 
> > > git fetch origin
> > > 
> > > git merge origin/main
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > Assuming there are no merge conflicts or any other issues. If you need more help feel free to reach out.
> > 
> > 
> > @rt4914 tbh i am having issue with my machines terminal and GitHub on it because it doesn't run git merge or git pull or push commands that well so i usually do it manually(either i download recent version and amend on local then refresh GitHub then paste the changes manually on GitHub remote or I just work on the files on GitHub directly all in my branch).
> > But overall i will fix this issue too.
> > Many thanks 😊
> 
> @KvngJamesNG Lets meet and try to fix it.

@rt4914 is it okay if we do it anytime today?

**Comment by rt4914** [comment]
> > > > @KvngJamesNG Also, please mention issue number in the Pull Request Description and in future follow the template that the pull request follows.
> > > > Actually you `main` and this current branch is not up-to-date with latest code. You will need to run commands like this in git-terminal to update your branches.
> > > > ### Updating local-main branch with the latest remote-main branch
> > > > ```
> > > > 
> > > > git checkout main
> > > > 
> > > > git fetch origin
> > > > 
> > > > git merge origin/main
> > > > ```
> > > > 
> > > > 
> > > >     
> > > >       
> > > >     
> > > > 
> > > >       
> > > >     
> > > > 
> > > >     
> > > >   
> > > > ### Updating current branch with main
> > > > ```
> > > > 
> > > > git checkout kvngjamesng/Add-Color-Palette-to-Website
> > > > 
> > > > git fetch origin
> > > > 
> > > > git merge origin/main
> > > > ```
> > > > 
> > > > 
> > > >     
> > > >       
> > > >     
> > > > 
> > > >       
> > > >     
> > > > 
> > > >     
> > > >   
> > > > Assuming there are no merge conflicts or any other issues. If you need more help feel free to reach out.
> > > 
> > > 
> > > @rt4914 tbh i am having issue with my machines terminal and GitHub on it because it doesn't run git merge or git pull or push commands that well so i usually do it manually(either i download recent version and amend on local then refresh GitHub then paste the changes manually on GitHub remote or I just work on the files on GitHub directly all in my branch).
> > > But overall i will fix this issue too.
> > > Many thanks 😊
> > 
> > 
> > @KvngJamesNG Lets meet and try to fix it.
> 
> @rt4914 is it okay if we do it anytime today?

For meeting related things discuss in group as I get notifications for that. I will be available from 3rd October so msg your time based on that in group.

File: tailwind.config.js (hunk 1 above)
Comment (KvngJamesNG) [reply tailwind.config.js]: this has been added

File: tailwind.config.js (hunk 2 above)
Comment (KvngJamesNG) [reply tailwind.config.js:12]: > If you check the resource that I shared https://stackoverflow.com/a/64175955
> 
> In that they use first get all the defaultcolors and than add the custom colors.
> 
> In you case you are directly adding custom colors and as a result we won't be able to use any color which tailwind provides by default.
> 
> Check the link again and try to apply solution similar to that.

done too

**Comment by KvngJamesNG** [comment]
> > @KvngJamesNG Also, please mention issue number in the Pull Request Description and in future follow the template that the pull request follows.
> > Actually you `main` and this current branch is not up-to-date with latest code. You will need to run commands like this in git-terminal to update your branches.
> > ### Updating local-main branch with the latest remote-main branch
> > ```
> > 
> > git checkout main
> > 
> > git fetch origin
> > 
> > git merge origin/main
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > ### Updating current branch with main
> > ```
> > 
> > git checkout kvngjamesng/Add-Color-Palette-to-Website
> > 
> > git fetch origin
> > 
> > git merge origin/main
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Assuming there are no merge conflicts or any other issues. If you need more help feel free to reach out.
> 
> @rt4914 tbh i am having issue with my machines terminal and GitHub on it because it doesn't run git merge or git pull or push commands that well so i usually do it manually(either i download recent version and amend on local then refresh GitHub then paste the changes manually on GitHub remote or I just work on the files on GitHub directly all in my branch).
> 
> But overall i will fix this issue too.
> 
> Many thanks 😊

it is now up to date I think

File: src/App.jsx [hunk 3]
```diff
@@ -1,8 +1,17 @@
 const App = () => {
   return (
+<<<<<<< HEAD
```
Comment (rt4914) [inline src/App.jsx:3]: This is becuse of merge conflict.

Change this entire code to return only this 

```
<h1 className="text-orange bg-brown text-4xl font-bold">
    Hello World!
</h1>
```

File: tailwind.config.js [hunk 4]
```diff
@@ -6,7 +8,18 @@ export default {
...
+        ...defaultColors,
+        'orange':{
+          DEFAULT: '#ff6725',
+        'dark': '#ff3f25',
+        'light': '#fef7f3',
+        },
+        'brown': '#453a34',
+        'green': '#00b579',
```
Comment (rt4914) [inline tailwind.config.js:20]: Indentation is incorrect. We should always match the brackets and lines correct. This is a common issues across multiple PRs so if you need more help in understanding this please message me.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@KvngJamesNG Suggested changes.

File: src/App.jsx (hunk 3 above)
Comment (KvngJamesNG) [reply src/App.jsx]: I have done this

File: tailwind.config.js (hunk 4 above)
Comment (KvngJamesNG) [reply tailwind.config.js]: I think I have done this one too

**Comment by KvngJamesNG** [comment]
@rt4914 I Reassigned it because there was no comment.

Kindly let me know if everything is okay.

File: src/App.jsx [hunk 5]
```diff
@@ -1,9 +1,5 @@
...
-    <>
-      <h1 className="underline p-4">Hello World!</h1>
-    </>
-  );
-};
-
-export default App;
+<h1 className="text-orange bg-brown text-4xl font-bold">
+  Hello World!
+</h1>
```
Comment (rt4914) [inline src/App.jsx:3]: This is incorrect. You still need `App` component in your code.

```
const App = () => {
  return (
    <>
      <h1 className="text-orange bg-brown text-4xl font-bold">Hello World!</h1>
    </>
  );
};

export default App;
```

Also, keep one blank line at the end of your code.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@KvngJamesNG PTAL

File: src/App.jsx (hunk 5 above)
Comment (KvngJamesNG) [reply src/App.jsx]: @rt4914 I have added whitespace and also return the App component in the code.. 
kindly check

**Review by rt4914 — APPROVED** [review]
LGTM, Thanks.

**Pull request merged**